libs/datasets/outlier_detection.py

```diff
@@ -14,8 +14,12 @@
 
 MultiRegionDataset = timeseries.MultiRegionDataset
 
-# Florida biweekly case reporting causes outlier detection to trigger inadvertently.
-STATES_TO_EXCLUDE_CASES_DEATHS = [RegionMask(AggregationLevel.COUNTY, states=["FL"])]
+STATES_TO_EXCLUDE_CASES_DEATHS = [
+    # FL: Biweekly case reporting causes outlier detection to trigger inadvertently.
+    # PA: Switching to weekly case reporting caused outlier detection to trigger inadvertently.
```

Comment:
Per the comment above, it's possible we actually don't need to do this.  But I'm fine either way. 🤷

Reply:
Yeah, agreed this probably isn't necessary--I'll remove all the blocks and create a snapshot to see what that looks like